core/templates/pages/exploration-editor-page/editor-tab/templates/modal-templates/customize-interaction-modal.controller.ts

```diff
@@ -158,16 +174,24 @@ angular.module('oppia').controller('CustomizeInteractionModalController', [
           InteractionDetailsCacheService.get(
             newInteractionId).customization);
       } else {
+        const customizationArgs = {};
         $scope.customizationArgSpecs.forEach(function(caSpec) {
-          StateCustomizationArgsService.displayed[caSpec.name] =
-            {
-              value: angular.copy(caSpec.default_value)
-            };
+          customizationArgs[caSpec.name] = (
+            InteractionCustomizationArgObjectFactory.createFromBackendDict(
+              {value: caSpec.default_value},
+              caSpec.schema
+            )
+          );
         });
+
+        StateCustomizationArgsService.displayed = customizationArgs;
       }
 
-      if (Object.keys(
-        StateCustomizationArgsService.displayed).length === 0) {
+      if (
+        Object.keys(
```

Comment:
Just as an FYI, the previous version was also fine. It's OK to break after any of the '('.